.github/ISSUE_TEMPLATE/technical-change.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Technical change
+about: Changes that do not affect the behaviour of the website. If you are not a member of the Technicie, there is a very high chance that you should not use this template
+title: ''
```

Comment:
`title: ''` looks a bit weird in the table. I believe the `title` can just be omitted here.